Bio/PDB/PDBList.py

```diff
@@ -412,52 +413,51 @@ def update_pdb(self, file_format=None, with_assemblies=False):
                     print(f"Obsolete file {old_file} is missing")
 
     def download_pdb_files(
-        self, pdb_codes, obsolete=False, pdir=None, file_format=None, overwrite=False
+        self,
+        pdb_codes: List[str],
```

Comment:
I think you can do just ``list[str]`` lower case without an import?

[Note we still support Python 3.8 in Biopython which might prevent that?]

Reply:
If I remember correctly, I encountered "type is not subscriptable" errors in the GitHub checks, which led me to use the types from the `typing` module.